README.md

```diff
@@ -61,6 +61,15 @@ will be updated. All users will be able to use your objects.
 
 ## How to Publish a Library
 
+Once the library is ready for publishing (i.e. all required changes are released)
+it can be published.
+
+### Pull recent changes from this repo
```

Comment:
@mximp don't use headings when the block of text inside the "section" consists of a single sentence. Headings are needed in order to separate more or less big blocks of text from each other. Here, without headings the text would look much more readable.

Reply:
fixed